src/nanorc/node.py

```diff
@@ -507,9 +507,26 @@ def _on_enter_callback(self, event):
                 done = []
                 for child_node in appset:
                     if not child_node.included: continue
-
-                    if not child_node.sup.desc.proc.is_alive() or not child_node.sup.commander.ping():
+                    IsAlive = child_node.sup.desc.proc.is_alive()
+                    if not IsAlive:
+                        failed.append(child_node.name)
+                        self.console.print(f"{child_node.name}, is dead")
+                        child_node.to_error(
+                            command = command,
+                        )
+                        done += [child_node]
+                        break
```

Comment:
shouldn't this be a `continue`?

Reply:
After more testing, I realised this may not be exactly what we want. Once one wib pod has failed should we stop trying to create the rest of the wib pods or should we we continue?

Reply:
The applications are created in the process manager. This is the application controller, which is just sending commands to applications, that happens a bit before in this file. This part of the code only waits for replies from applications.

Reply:
If I use continue, as soon as one pod fails to boot, they all stop booting automatically and an error is thrown (New Logic). If I use break, the rest of the pods continue even if one is already broken which could mean I have failure in one or two pods but the other pods are fine, alive and pingable. Which logic is better for our use case?

Reply:
If I'm not explaining this well we can talk about it in our meeting tomorrow

Reply:
![image](https://github.com/DUNE-DAQ/nanorc/assets/123565780/74d16c63-9e42-430d-8359-9758d645102f)
![image](https://github.com/DUNE-DAQ/nanorc/assets/123565780/2c59623c-ee38-4ebf-970c-192bd13053f7)

In one none of them conf because one fails, in the other one the rest are able to conf even if one does not